build.properties

```diff
@@ -72,6 +72,7 @@ javadoc.header=JavaFX&nbsp;13
 #
 ##############################################################################
 
+# JDK
 jfx.build.jdk.version.min=11
```

Comment:
Should I change this to 12 & build number to 33?

Reply:
No. The minimum is still JDK 11. Instead, you can reintroduce the `jfx.build.jdk.version` and `jfx.build.jdk.buildnum` variables to indicate the version that we use in production.